src/main/java/dk/cngroup/trainings/spring/springassignment/config/WebMvcConfig.java

```diff
@@ -0,0 +1,16 @@
+package dk.cngroup.trainings.spring.springassignment.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebMvcConfig implements WebMvcConfigurer {
+
+	@Bean
```

Comment:
is there any particular reason why this is not in the same Security config?

Reply:
I just didn't think it applies any security regarding the application flow.
But I think I will move it there since the Security class isn't that big for now.

Reply:
Password encoder definately applies to seurity :)